server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -482,6 +486,11 @@ export function configureWebSocketServices(
                             "signal",
                             createRoomJoinMessage(message.connection.clientId, message.details));
                     }
+                    // excluding summarizer for total client count.
+                    if (message?.details?.details?.type !== "summarizer") {
```

Comment:
Just for learning, what other types? Maybe we should have an allowed types instead?
And could type case sensitive

Reply:
we want to include all clients except summarizer, to be able to calculate actual clients aka interactive